server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -173,6 +262,38 @@ export function verifyStorageToken(
 				new NetworkError(403, "Missing documentId in request"),
 			);
 		}
+
+		// TODO: remove this check and code after this block after validation
```

Comment:
Are you going to do a follow-up PR to do this? And will that include removing `singleUseTokenCache` option?

Reply:
Plan to do it in follow up PR and remove singleUseTokenCache after some time.